src/Projections/Windows.UI.Xaml/Windows.UI.Xaml.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(LibBuildTFMsNoNetStandard)</TargetFrameworks>
+    <Platforms>x64;x86</Platforms>
+    <AssemblyName>Microsoft.Windows.UI.Xaml</AssemblyName>
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Numerics.Vectors" Version="4.5.0" />
```

Comment:
Can we condition this so it's skipped when on TFMs that already have these APIs?

Reply:
This projection is just for testing. I'll take a look at cleaning this up, but I don't see it as high-priority.